.deps/EXCLUDED/prod.md

```diff
@@ -0,0 +1,6 @@
+This file lists dependencies that do not need CQs or auto-detection does not work due to a bug in https://github.com/eclipse/dash-licenses
+| Packages | Resolved CQs |
+| --- | --- |
+| `che-dashboard-e2e@1.0.0` | - |
```

Comment:
I propose to set `N/A` here and below to make it clearer that CQs is not applicable here since they are both eclipse projects. WDYT?

Reply:
I totally agree with you.

Reply:
I have changed it.